src/components/Drawer.js

```diff
@@ -158,6 +158,13 @@ export default function Drawer(props) {
     }
   }, []);
 
+  const dimensions = {
+    width: '100%',
+    height: '100vh',
+    top: 0,
+    left: 0,
+  };
+
```

Comment:
@okdv do you have to add this to drawer? is it possible to put it somewhere else? The drawer is pretty fragile I feel, I prefer to sacrify something to keep it simple

Reply:
Oh, I guess the code in drawer is to solve issue #1039 , could you slit it to two PR for these two respectively?

Reply:
@dadiorchen didn't actually test it without those settings, since I had just grabbed them from the existing style of the drawer. But I can look into seeing if something if its necessary?